src/reality-loader/live_video.ts

```diff
@@ -8,24 +8,17 @@ import { VuforiaServiceDelegate } from '../vuforia'
 export class LiveVideoRealityLoader extends RealityLoader {
     public type = 'live-video';
 
-    public iframeElement: HTMLIFrameElement;
     public videoElement: HTMLVideoElement;
 
     constructor(
             private sessionService: SessionService,
             private vuforiaDelegate: VuforiaServiceDelegate) {
         super();
 
-        this.iframeElement = document.createElement('iframe');
-        this.iframeElement.style.border = '0';
-        this.iframeElement.width = '100%';
-        this.iframeElement.height = '100%';
-
         this.videoElement = document.createElement('video');
-        this.videoElement.width = this.iframeElement.width;
-        this.videoElement.height = this.iframeElement.height;
+        this.videoElement.width = '100%';
+        this.videoElement.height = '100%';
         this.videoElement.controls = false;
```

Comment:
Make sure this code only runs in a DOM environment (wrap it in an if statement that checks for the existence of the document object)